mypy.ini

```diff
@@ -6,4 +6,4 @@ plugins =
     mypy_drf_plugin.main
 
 [mypy.plugins.django-stubs]
-django_settings_module = stade.settings.development
+django_settings_module = stade.settings.testing
```

Comment:
I wonder if just setting this as an env var in tox.ini is good enough? Or do these stubs need it specified here?

Reply:
It's not currently set in the Tox file, and I think it actually is going to be easier to maintain here. If we ever want to run MyPy directly, this is one fewer thing to configure manually.